crates/fluvio-types/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fluvio-types"
-version = "0.5.0"
+version = "0.6.0"
```

Comment:
no need to bump to `0.6.0` since this doesn't change existing api.   Can bump to `0.5.1`

Reply:
Of course!